lib/galaxy/config/schemas/config_schema.yml

```diff
@@ -386,7 +386,8 @@ mapping:
         required: false
```

Comment:
I think we should remove the default from the line above like we do for `managed_config_dir` and `data_dir`.